tests/wpt/web-platform-tests/html/semantics/forms/the-input-element/type-change-state.html

```diff
@@ -51,6 +51,8 @@
             input.type = types[j].type;  // change state
             if (types[j].sanitizedValue || types[j].sanitizedValue === "") {
               assert_equals(input.value, types[j].sanitizedValue, "input.value should be " + types[j].sanitizedValue + " after change of state");
+            } else if (types[i].sanitizedValue || types[i].sanitizedValue === "") {
```

Comment:
What is this?

Reply:
So this test is basically switching between various different input types for an input element. When you set an input's value OR change the input type, the value sanitization algorithm would run. This added conditional here is used to check whether the previous input type had a sanitized value, and if so, use it on the assert_equals comparison below.

Reply:
This would also explain the new expected tests failures, because our value sanitization algorithm is incomplete.